src/main/java/com/epam/page/object/generator/validators/WebValidators.java

```diff
@@ -0,0 +1,50 @@
+package com.epam.page.object.generator.validators;
+
+import com.epam.page.object.generator.models.WebPage;
+import com.epam.page.object.generator.models.webgroup.WebElementGroup;
+import com.epam.page.object.generator.validators.web.ElementUniquenessValidator;
+import com.epam.page.object.generator.validators.web.UniquenessAttributeExistenceValidator;
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+import org.assertj.core.util.Lists;
+
+public class WebValidators {
```

Comment:
I would suggest to merge WebValidators and JsonValidators into single class BatchValidator.
Add 2 static methods BatchValidator that create instances of BatchValidator with web and json validators respectively.

Reply:
I think that we need to stay WebValidator and JsonValidator, maybe they are really similar, but when they in the different classes they have suitable api. We easily can add new validator in those type which we need. If we will need to add more logic in JsonValidator we can easily add it in JsonValidator class and we won't have one big class BatchValidator which will contains all about all validators.